src/remote/server/os/posix/inet_server.cpp

```diff
@@ -314,6 +315,10 @@ int CLIB_ROUTINE main( int argc, char** argv)
 		// activate paths set with -e family of switches
 		ISC_set_prefix(0, 0);
 
+		// set shutdown signals handler for listener
```

Comment:
Does it make sense to add this handlers in super & superclassic modes?

Reply:
No, it's only relevant to classic

Reply:
I'll add a condition